packages/ccda/src/fhir-to-ccda.ts

```diff
@@ -918,26 +930,20 @@ class FhirToCcdaConverter {
    * @param identifiers - The FHIR identifiers to map.
    * @returns The C-CDA identifiers.
    */
-  private mapIdentifiers(
-    id: string | undefined,
-    identifiers: (Identifier | undefined)[] | undefined
-  ): CcdaId[] | undefined {
+  private mapIdentifiers(id: string | undefined, identifiers: Identifier[] | undefined): CcdaId[] | undefined {
     const result: CcdaId[] = [];
 
     if (id) {
-      result.push({
-        '@_root': id,
-      });
+      result.push({ '@_root': id });
     }
 
     if (identifiers) {
       for (const id of identifiers) {
-        if (id) {
-          result.push({
-            '@_root': mapFhirSystemToCcda(id.system),
-            '@_extension': id.value,
-          });
+        const root = mapFhirSystemToCcda(id.system);
```

Comment:
FHIR `Identifier` array can support any arbitrary system as URL.

C-CDA equivalent only supports predefined OID "roots".

We have the mapping for many of the common identifiers/roots, such as NPI, US Drivers License, US SSN, etc, but if you have `system: 'https://example.com` in your FHIR resource, this would end up with an empty root, which is invalid.